src/prog_algs/metrics/uncertain_data_metrics.py

```diff
@@ -0,0 +1,104 @@
+# Copyright © 2021 United States Government as represented by the Administrator of the National Aeronautics and Space Administration. All Rights Reserved.
+
+"""
+This file includes functions for calculating general metrics on uncertain data
```

Comment:
Maybe a bit more information here, something like "This file includes functions for calculating general metrics (i.e. mean, std, percentiles, etc.) on any distribution (e.g. states, event_states, an EOL distribution, etc.)"

Reply:
```suggestion
This file includes functions for calculating general metrics (i.e. mean, std, percentiles, etc.) on any distribution of type UncertainData (e.g. states, event_states, an EOL distribution, etc.)
```